src/components/Menus.tsx

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Create an interface for the items. 
something like this: `export interface SomeName{
    name: string;
    link: string
}`

and then use as: `export const Company: SomeName[] = [
    {
        name: "...",
        link: "..."
    }
]`

This helps with strongly typing stuff